site/content/docs/dashboards-v2/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Overview
+weight: 47
+slug: /
+menu:
+  docs:
+    parent: "Dashboards V2"
+    identifier: overview-dashboards-v2
+---
+
+> These are the docs for the ✨ **new Checkly Dashboards** ✨, available for all users by enabling them in the [Labs section under account settings](https://app.checklyhq.com/settings/account/labs).
+> Docs for the "old" dashboards can be found [here](/docs/dashboards/)
+
+## Overview
+
+![Dashboard example](/docs/images/dashboards-v2/public-dashboard-1.png)
+
+Checkly dashboards are public dashboards available on every plan. Use them as a status page for your app or service  or as
+a dashboard on a wall mounted TV in the office. Dashboards allow you to do the following:
```

Comment:
Maybe `They allow you to do the following:` to avoid repeating `Dashboards` again